modules/social_features/social_event/src/Service/SocialEventEnrollServiceInterface.php

```diff
@@ -19,6 +19,10 @@ interface SocialEventEnrollServiceInterface {
    *
    * @return bool
    *   TRUE if enrollment is allowed.
+   *
+   * @deprecated in social:11.4.0 and is removed from social:12.0.0. Use
```

Comment:
@nechai Please change version from 11.4 to 11.5

Reply:
Added